include/envoy/http/codec.h

```diff
@@ -147,7 +147,7 @@ class ResponseEncoder : public virtual StreamEncoder {
    * Indicates whether invalid HTTP messaging should be handled with a stream error or a connection
    * error; if value is missing, then HCM drives behaviour on invalid HTTP messaging.
```

Comment:
update comment to remove "if value is missing, then HCM drives behaviour on invalid HTTP messaging."